extensions/amp-story-audio-sticker/0.1/amp-story-audio-sticker.css

```diff
@@ -0,0 +1,12 @@
+amp-story-audio-sticker {
+  z-index: 3 !important;
+  pointer-events: visible !important;
```

Comment:
```suggestion
  pointer-events: auto !important;
```
`visible` is for SVG elements:

<img width="393" alt="image" src="https://user-images.githubusercontent.com/22420856/226374418-d12f03a4-dcd6-4893-92cb-04bcd9faca9d.png">